primitives/phragmen/src/lib.rs

```diff
@@ -85,7 +101,14 @@ pub struct Edge<AccountId> {
 pub type PhragmenAssignment<AccountId> = (AccountId, Perbill);
 
 /// Means a particular `AccountId` was backed by `Votes` of a nominator's stake.
-pub type PhragmenStakedAssignment<AccountId> = (AccountId, Votes);
+#[derive(RuntimeDebug)]
```

Comment:
Votes, Power and Balances might have different meaning, need double checking here if we are goting to replace Votes.

Reply:
Yes. 

Total power is `1_000_000_000u32`.
The votes is `u64` which use for electing. There're some `mul` in electing which will overflow `Power: u32`. The `DEN` is `u64::max_value()`, if we use `1_000_000_000` there will be some accuracy problem (you can change it to `1_000_000_000` and run `cargo test` in phragmen folder). 
The balance is a cached data for slashing.